<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlStationManageWindow</class>
 <widget class="QWidget" name="ControlStationManageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>无人平台控制站管理</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    background-color: #f5f7fa;
    font-family: "Microsoft YaHei", "Segoe UI", sans-serif;
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QWidget" name="buttonContainer" native="true">
     <property name="styleSheet">
      <string notr="true">
       background-color: white;
       border-radius: 6px;
       padding: 8px;
       box-shadow: 0 1px 4px rgba(0,0,0,0.05);
      </string>
     </property>
     <layout class="QHBoxLayout" name="buttonLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              background-color: #3a6ea5;
              color: white;
              border: none;
              padding: 8px 16px;
              border-radius: 4px;
              font-size: 13px;
              min-width: 80px;
              transition: background-color 0.2s;
          }
          QPushButton:hover {
              background-color: #4a90e2;
              box-shadow: 0 2px 4px rgba(0,0,0,0.1);
          }
          QPushButton:pressed {
              background-color: #2a5685;
          }
          QPushButton:disabled {
              background-color: #cccccc;
          }
         </string>
        </property>
        <property name="text">
         <string>新增控制站</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEdit">
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              background-color: #3a6ea5;
              color: white;
              border: none;
              padding: 8px 16px;
              border-radius: 4px;
              font-size: 13px;
              min-width: 80px;
          }
          QPushButton:hover {
              background-color: #4a90e2;
          }
          QPushButton:pressed {
              background-color: #2a5685;
          }
         </string>
        </property>
        <property name="text">
         <string>编辑</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/edit.png</normaloff>:/icons/edit.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              background-color: #e74c3c;
              color: white;
              border: none;
              padding: 8px 16px;
              border-radius: 4px;
              font-size: 13px;
              min-width: 80px;
          }
          QPushButton:hover {
              background-color: #c0392b;
          }
          QPushButton:pressed {
              background-color: #992d22;
          }
         </string>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineSearch">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
          QLineEdit {
              padding: 8px 12px;
              border: 1px solid #d0d0d0;
              border-radius: 4px;
              font-size: 13px;
              min-width: 250px;
              background-color: white;
              color: #333333;
          }
          QLineEdit:focus {
              border: 1px solid #4a90e2;
              box-shadow: 0 0 0 2px rgba(74,144,226,0.2);
          }
          QLineEdit::placeholder {
              color: #999999;
              font-style: italic;
          }
         </string>
        </property>
        <property name="placeholderText">
         <string>输入控制站名称、IP等关键词...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSearch">
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              background-color: #27ae60;
              color: white;
              border: none;
              padding: 8px 16px;
              border-radius: 4px;
              font-size: 13px;
              min-width: 80px;
          }
          QPushButton:hover {
              background-color: #2ecc71;
          }
          QPushButton:pressed {
              background-color: #219955;
          }
         </string>
        </property>
        <property name="text">
         <string>搜索</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableStations">
     <property name="styleSheet">
      <string notr="true">
       QTableWidget {
           font-size: 13px;
           background-color: white;
           alternate-background-color: #f8f9fa;
           border: 1px solid #e0e0e0;
           border-radius: 6px;
           gridline-color: #e0e0e0;
           outline: 0;
       }
       QTableWidget::item {
           padding: 8px;
           border-bottom: 1px solid #f0f0f0;
           color: #333333;
       }
       QTableWidget::item:hover {
           background-color: #f0f7ff;
       }
       QHeaderView::section {
           background-color: #3a6ea5;
           color: white;
           padding: 8px;
           border: none;
           font-weight: 500;
           font-size: 13px;
       }
       QHeaderView {
           border-top-left-radius: 6px;
           border-top-right-radius: 6px;
       }
      </string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>8</number>
     </property>
     <column>
      <property name="text">
       <string/>
      </property>
      <property name="resizeMode">
       <enum>QHeaderView::Fixed</enum>
      </property>
      <property name="minimumWidth">
       <number>40</number>
      </property>
      <property name="maximumWidth">
       <number>40</number>
      </property>
     </column>
     <column>
      <property name="text">
       <string>控制站名称</string>
      </property>
      <property name="resizeMode">
       <enum>QHeaderView::Stretch</enum>
      </property>
     </column>
     <column>
      <property name="text">
       <string>所属单位</string>
      </property>
      <property name="resizeMode">
       <enum>QHeaderView::Interactive</enum>
      </property>
     </column>
     <column>
      <property name="text">
       <string>控制站类型</string>
      </property>
      <property name="resizeMode">
       <enum>QHeaderView::ResizeToContents</enum>
      </property>
     </column>
     <column>
      <property name="text">
       <string>IP地址</string>
      </property>
      <property name="resizeMode">
       <enum>QHeaderView::ResizeToContents</enum>
      </property>
     </column>
     <column>
      <property name="text">
       <string>通信端口</string>
      </property>
      <property name="resizeMode">
       <enum>QHeaderView::ResizeToContents</enum>
      </property>
     </column>
     <column>
      <property name="text">
       <string>状态</string>
      </property>
      <property name="resizeMode">
       <enum>QHeaderView::ResizeToContents</enum>
      </property>
     </column>
     <column>
      <property name="text">
       <string>更新时间</string>
      </property>
      <property name="resizeMode">
       <enum>QHeaderView::ResizeToContents</enum>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="paginationWidget" native="true">
     <property name="styleSheet">
      <string notr="true">
       background-color: white;
       border-radius: 6px;
       padding: 8px;
       box-shadow: 0 1px 4px rgba(0,0,0,0.05);
      </string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="labelPageInfo">
        <property name="styleSheet">
         <string notr="true">
         color: #333333;
         font-size: 13px;
         font-weight: 500;
         background-color: transparent;
         </string>
        </property>
        <property name="text">
         <string>第 1 页 / 共 1 页 (10 条)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnFirst">
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              min-width: 60px;
              padding: 6px 8px;
              background-color: #f8f9fa;
              border: 1px solid #d0d0d0;
              border-radius: 4px;
              font-size: 12px;
          }
          QPushButton:hover {
              background-color: #e9ecef;
          }
          QPushButton:pressed {
              background-color: #dee2e6;
          }
         </string>
        </property>
        <property name="text">
         <string>首页</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/first.png</normaloff>:/icons/first.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPrev">
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              min-width: 60px;
              padding: 6px 8px;
              background-color: #f8f9fa;
              border: 1px solid #d0d0d0;
              border-radius: 4px;
              font-size: 12px;
          }
          QPushButton:hover {
              background-color: #e9ecef;
          }
          QPushButton:pressed {
              background-color: #dee2e6;
          }
         </string>
        </property>
        <property name="text">
         <string>上一页</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/prev.png</normaloff>:/icons/prev.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboPage">
        <property name="styleSheet">
         <string notr="true">
          QComboBox {
              padding: 4px 8px;
              border: 1px solid #d0d0d0;
              border-radius: 4px;
              min-width: 60px;
              font-size: 12px;
              padding: 4px 8px;
              color: #333333;
              background-color: white;
          }
          QComboBox::drop-down {
              width: 20px;
              border: none;
          }
          QComboBox QAbstractItemView {
              background-color: white;
              color: #333333;
              selection-background-color: #3a6ea5;
              selection-color: white;
          }
         </string>
        </property>
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNext">
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              min-width: 60px;
              padding: 6px 8px;
              background-color: #f8f9fa;
              border: 1px solid #d0d0d0;
              border-radius: 4px;
              font-size: 12px;
          }
          QPushButton:hover {
              background-color: #e9ecef;
          }
          QPushButton:pressed {
              background-color: #dee2e6;
          }
         </string>
        </property>
        <property name="text">
         <string>下一页</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/next.png</normaloff>:/icons/next.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLast">
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              min-width: 60px;
              padding: 6px 8px;
              background-color: #f8f9fa;
              border: 1px solid #d0d0d0;
              border-radius: 4px;
              font-size: 12px;
          }
          QPushButton:hover {
              background-color: #e9ecef;
          }
          QPushButton:pressed {
              background-color: #dee2e6;
          }
         </string>
        </property>
        <property name="text">
         <string>末页</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/last.png</normaloff>:/icons/last.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboPageSize">
        <property name="styleSheet">
         <string notr="true">
          QComboBox {
              padding: 4px 8px;
              border: 1px solid #d0d0d0;
              border-radius: 4px;
              min-width: 80px;
              font-size: 12px;
              color: black;
          }
          QComboBox::drop-down {
              width: 20px;
              border: none;
          }
          QComboBox QAbstractItemView {
              color: black;
              background-color: white;
          }
         </string>
        </property>
        <item>
         <property name="text">
          <string>10 条/页</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20 条/页</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50 条/页</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100 条/页</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
